### DISCUSSION TITLE
Copilot generated commit messages

### DISCUSSION THREAD

**Discussion by WillTheDeveloper** [community/community#20865] [Copilot Conversations]
If co-pilot is able to understand the context of the code that is being changed then it would be pretty epic if it was able to automatically generate commit messages inside the IDE when you got to commit your changes.

**Comment by AnuraghSarkar** [comment]
Yeah that would be awesome and what;s awesome now, code comment is generated by copilot wow!

**Comment by so1ve** [comment]
It should work now!
